raster-foundry/raster-foundry
# Fix empty exported annotation shapefile bug

## Overview

This PR fixes the annotation export as shapefile bug so that the exported shapefile is not empty.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

 * Run migration
 * Go to a project with annotations, if not, create some annotations.
 * Export as a shapefile.
 * Open the shapefile and make sure it is not empty.

Closes #3608
